public/src/components/Main.tsx

```diff
@@ -41,10 +41,10 @@ export interface MainProps {
     stocksReservedMap: {[index: number]: number} //stocks reserved from user for a given stockid
     constantsMap:    { [index:string]: number } // various constants. Documentation found in server/actionservice/Login method
 
-    marketIsOpenHackyNotif: 	string
-    marketIsClosedHackyNotif: 	string
     isMarketOpen: 				boolean
     isPhoneVerified:            boolean
+    isBlocked:                  boolean
```

Comment:
Why is `isBankrupt` a state variable in both Ap.tsx and Main.tsx? Make it a state variable in one and pass it as props in another.